hospital_has_doctor.py

```diff
@@ -0,0 +1,17 @@
+class Doctor:
+    hospt_name="appolo"
+    def __init__(self,doctor_name,specialization):
+        self.doctor_name=doctor_name
+        self.specialization=specialization
+    def hospt(self):
+        print("hospital is located in nrt")
+class hospital:
```

Comment:
naming convention for classes is recommended as Title case   which Hopsital rather than hospital

Reply:
ok sir, i will change it

Reply:
Sir, I change the mistakes.Please approve it